### PR TITLE
Add build scripts and docker files to build .debs for various ubuntu versions with or without -rpgp

### PR DESCRIPTION
**Pull request by Jikstra**
Things missing:
- [ ] wrapper for rpgp script doesn't pass down arguments
- [ ] do a general build-image/build-package.sh
- [ ] the install .deb files have some wrong/missing metadata:
  - [ ] app is marked as proprietary
  - [ ] no icon on install
  - [ ] make it overall nicer/more appealing
- [ ] circleci integration & automation

### CODE REVIEW

**Comment by Jikstra** [comment]
So this currently is able to build .deb files for ubuntu16.04 and ubuntu18.10. 
Short instruction:
```
# cd to the folder where this branch in

# creates/builds the ubuntu 16.04 docker image
./builder/ubuntu-16_04/build-image.sh

# execute the packaging script in current folder ("." will be mounted into the docker image)
# CAUTION: this will rm -rf node_modules and install it again. So this might break your node modules, to make it work on your host again you might need to rm -rf node_modules and download them again.
./builder/ubuntu-16_04/build-package.sh

# You should  have a .deb in ./dist/ubuntu-16_04/ folder
# Same procedure with 18.10

, just use the scripts from the 18.10 folder.
```

File: builder/ubuntu-18_10-rpgp/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    && curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly -y
+
+ENV PATH=/root/.cargo/bin:$PATH
+
+# Install rpgp
+RUN true \
+    && cd / \
+    && git clone https://github.com/Jikstra/rpgp.git \
+    && cd rpgp/pgp-ffi \
+    && git checkout fix_makefile_pkgconfig_typo \
```
Comment (hpk42) [inline builder/ubuntu-18_10-rpgp/docker/Dockerfile]: not needed anymore.

File: builder/ubuntu-18_10-rpgp/docker/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+RUN true \
+    && apt -y install curl \
+    && curl https://sh.rustup.rs -sSf | sh -s -- --default-toolchain nightly -y
+
+ENV PATH=/root/.cargo/bin:$PATH
+
+# Install rpgp
+RUN true \
+    && cd / \
+    && git clone https://github.com/Jikstra/rpgp.git \
```
Comment (hpk42) [inline builder/ubuntu-18_10-rpgp/docker/Dockerfile]: why from your fork?

Comment (Jikstra) [reply builder/ubuntu-18_10-rpgp/docker/Dockerfile]: Not needed any more, changes got merged

**Comment by Jikstra** [comment]
So i uploaded test -rpgp builds for ubuntu 16.04, 18.04 and 18.10. They seem to work in my virtual boxes, please try them out and stress test them a bit :)

https://github.com/deltachat/deltachat-desktop/releases/tag/v0.102.0-dev-rpgp

**Comment by Jikstra** [comment]
Superseeded by https://github.com/deltachat/deltachat-desktop/pull/752

**Pull request closed without merging**